### ISSUE TITLE
Normalize with imagenet stats

### ISSUE THREAD

**Issue by jaideep11061982** [rwightman/efficientdet-pytorch#61]
Do we need to normalize the image with imagnet stats while we are training with custom dataset ?
i see alex in his public kernel using effdet is not doing so, i was wondering if that is doen some where in your model framework

**Comment by rwightman** [comment]
@jaideep11061982 no, you do not, it's the default for the pretrained weights and when training the detector with an imagenet pretrained backbone that uses imagenet mean. Coco and Imagenet means will be quite close. 

For custom datasets I would use different values if they differ significantly. The only scenario I'd reconsider that is if you're freezing the batch norms extensively. 

For the training script here, you'd change it by pass an arg to the create_loader call, it is left default right now https://github.com/rwightman/efficientdet-pytorch/blob/master/data/loader.py#L86

**Comment by jaideep11061982** [comment]
Hi rwiightman 
thanks for quick reply. I think we are not calling in your create_Data_loader for our custom Data set .. Dataloader is created using Std pytorch API. In that case should we have additional Aug that would do normalize?

Secondly,
 net = EfficientDet(config, pretrained_backbone=False)
Is it advisable to use pretrained_backbon=True or False.Below is piece of code i use for building net.

```
def get_net():
    config = get_efficientdet_config('tf_efficientdet_d5')
    #print(config)
    net = EfficientDet(config, pretrained_backbone=False)
    checkpoint = model_zoo.load_url('https://github.com/rwightman/efficientdet-pytorch/releases/download/v0.1/tf_efficientdet_d5-ef44aea8.pth')
    #('../input/efficientdet/efficientdet_d5-ef44aea8.pth') /home/temp/.cache/torch/checkpoints/tf_efficientdet_d5-ef44aea8.pth
    net.load_state_dict(checkpoint)
    config.num_classes = 1
    config.image_size = 512
    net.class_net = HeadNet(config, num_outputs=config.num_classes, norm_kwargs=dict(eps=.001, momentum=.01))
    return DetBenchTrain(net, config)
```

**Comment by rwightman** [comment]
If you're not using the loader setup here, yeah, do the normalize like you would for any other pytorch transform pipeline. Don't need pretrained backbone if you're loading from pretrained detection model weights, so could set it to false.
